sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ChangeFeedFetcher.java

```diff
@@ -120,23 +119,23 @@ public ChangeFeedFetcher(
 
     private Mono<FeedResponse<T>> nextPageInternal() {
         return Mono.fromSupplier(this::nextPageCore)
-                   .flatMap(Function.identity())
-                   .flatMap((r) -> {
-                       FeedRangeContinuation continuationSnapshot =
-                           this.changeFeedState.getContinuation();
-
-                       if (continuationSnapshot != null &&
-                           continuationSnapshot.handleChangeFeedNotModified(r) == ShouldRetryResult.RETRY_NOW) {
-
-                           // not all continuations have been drained yet
-                           // repeat with the next continuation
-                           this.reenableShouldFetchMoreForRetry();
-                           return Mono.empty();
-                       }
-
-                       return Mono.just(r);
-                   })
-                   .repeatWhenEmpty(o -> o);
+            .flatMap(Function.identity())
+            .flatMap((r) -> {
+                FeedRangeContinuation continuationSnapshot =
+                    this.changeFeedState.getContinuation();
+
+                if (continuationSnapshot != null &&
+                    continuationSnapshot.handleChangeFeedNotModified(r) == ShouldRetryResult.RETRY_NOW) {
+
+                    // not all continuations have been drained yet
+                    // repeat with the next continuation
+                    this.reenableShouldFetchMoreForRetry();
+                    return Mono.empty();
+                }
+
+                return Mono.just(r);
+            })
+            .repeatWhenEmpty(o -> o);
```

Comment:
Looks like only formatting change, so better revert this

Reply:
done